app/assets/javascripts/modules/was_seed_viewer.js

```diff
@@ -1,9 +1,34 @@
+/*global Sly */
 
 (function( global ) {
   'use strict';
   var Module = (function() {
+
     return {
       init: function() {
+        var $wasThumbFrame = $('.sul-embed-was-seed');
+        var dataAttributes = $wasThumbFrame.data();
+
+        // Create the SLY scroll div on fly
+        var $wasThumbScroll = $('<div class="sul-embed-thumb-slider-scroll"><div class="sul-embed-thumb-slider-handle"/></div>');
```

Comment:
We don't seem to be testing for this scroll bar anywhere (if removed our tests would still pass). Should we add some js tests here?